Lib/enum.py

```diff
@@ -800,25 +800,18 @@ def __call__(cls, value, names=None, *, module=None, qualname=None, type=None, s
                 )
 
     def __contains__(cls, member):
-        """
-        Return True if member is a member of this enum
-        raises TypeError if member is not an enum member
+        """Return True if `member` is in `cls`.
+
+        `member` is in `cls` iff:
+        1) `member` is a proper member of the `cls` enum, or
+        2) `member` is the value of a member of the `cls` enum.
 
-        note: in 3.12 TypeError will no longer be raised, and True will also be
-        returned if member is the value of a member in this enum
+        Beware that 2) can lead to some confussion if members of different
```

Comment:
```suggestion
        Beware that 2) can lead to some confusion if members of different
```

Reply:
Thank you, I have corrected the spelling mistake.